StRoot/StEvent/StFwdTrack.h

```diff
@@ -167,11 +173,11 @@ class StFwdTrack : public StObject {
     float mNDF;
     float mPval;
     short mCharge;
-    StThreeVectorD mPrimaryMomentum;
+    StThreeVectorD mPrimaryMomentum; 
```

Comment:
Memory layout changes, ClassDef should be version 3.

Reply:
See comment below, all changes related to FTT reverted since they should be handled in PR #569 .
Done with https://github.com/star-bnl/star-sw/pull/571/commits/6088f974a30af0141f460b3fdd3183ae4f7c66ec